test/TestModel10_simulationItermethods.py

```diff
@@ -0,0 +1,42 @@
+'''
```

Comment:
@michaeltryby,  The following illustrates three approaches to run a simulation with pyswmm.